cmd/runtimetest/main.go

```diff
@@ -628,11 +663,16 @@ func validate(context *cli.Context) error {
 	t := tap.New()
 	t.Header(0)
 
+	complianceLevelString := context.String("compliance-level")
+	complianceLevel := ociErr.ParseLevel(complianceLevelString)
 	var validationErrors error
 	for _, v := range defaultValidations {
 		err := v.test(spec)
 		t.Ok(err == nil, v.description)
 		if err != nil {
+			if e, ok := err.(*ociErr.Error); ok && e.Level < complianceLevel {
+				continue
```

Comment:
We may want to print these as warnings even if they are non-fatal.  With TAP, I'd recommend using [diagnostics][1].  I'm not sure what the best approach is with Go's testing framework, but we should probably figure that out ;).

[1]: https://testanything.org/tap-version-13-specification.html#diagnostics